### PR TITLE
FLAG-75: refactor dao method with stream API

### PR DESCRIPTION
**Pull request by ManojLL**
issue worked on : [FLAG-75](https://openmrs.atlassian.net/browse/FLAG-75)

[FLAG-75]: https://openmrs.atlassian.net/browse/FLAG-75?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: api/src/main/java/org/openmrs/module/patientflags/db/hibernate/HibernateFlagDAO.java [hunk 1]
```diff
@@ -409,9 +409,9 @@ public void deletePatientFlagsForPatient(Patient patient) throws DAOException {
 		
 		@SuppressWarnings("unchecked")
 		List<PatientFlag> flags = criteria.list();
-		for (PatientFlag patientFlag : flags) {
+		flags.forEach(patientFlag->{
```
Comment (wikumChamith) [inline api/src/main/java/org/openmrs/module/patientflags/db/hibernate/HibernateFlagDAO.java:412]: Format the code like this:
`flags.forEach(patientFlag -> {`

File: api/src/main/java/org/openmrs/module/patientflags/db/hibernate/HibernateFlagDAO.java [hunk 2]
```diff
@@ -426,9 +426,9 @@ public void deletePatientFlagForPatient(Patient patient, Flag flag) throws DAOEx
 		
 		@SuppressWarnings("unchecked")
 		List<PatientFlag> flags = criteria.list(); //Should return a maximum of one flag
-		for (PatientFlag patientFlag : flags) {
+		flags.forEach(patientFlag->{
```
Comment (wikumChamith) [inline api/src/main/java/org/openmrs/module/patientflags/db/hibernate/HibernateFlagDAO.java:429]: same here

File: api/src/main/java/org/openmrs/module/patientflags/db/hibernate/HibernateFlagDAO.java [hunk 3]
```diff
@@ -439,12 +439,12 @@ public void deletePatientFlagsForFlag(Flag flag) throws DAOException {
 		Criteria criteria = sessionFactory.getCurrentSession().createCriteria(PatientFlag.class);
 		criteria.add(Restrictions.eq("flag", flag));
 		criteria.add(Restrictions.eq("voided", false));
-		
+
 		@SuppressWarnings("unchecked")
 		List<PatientFlag> flags = criteria.list();
-		for (PatientFlag patientFlag : flags) {
+		flags.forEach(patientFlag->{
```
Comment (wikumChamith) [inline api/src/main/java/org/openmrs/module/patientflags/db/hibernate/HibernateFlagDAO.java:445]: same here

File: api/src/main/java/org/openmrs/module/patientflags/db/hibernate/HibernateFlagDAO.java (hunk 1 above)
Comment (ManojLL) [reply api/src/main/java/org/openmrs/module/patientflags/db/hibernate/HibernateFlagDAO.java]: I formatted the code by adding spaces

File: api/src/main/java/org/openmrs/module/patientflags/db/hibernate/HibernateFlagDAO.java (hunk 2 above)
Comment (ManojLL) [reply api/src/main/java/org/openmrs/module/patientflags/db/hibernate/HibernateFlagDAO.java]: I formatted the code by adding spaces

**Pull request merged**